nullaway/src/test/java/com/uber/nullaway/NullAwayKeySetIteratorTests.java

```diff
@@ -0,0 +1,132 @@
+package com.uber.nullaway;
+
+import org.junit.Test;
+
+public class NullAwayKeySetIteratorTests extends NullAwayTestsBase {
+
+  @Test
+  public void mapKeySetIteratorBasic() {
+    defaultCompilationHelper
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import java.util.*;",
```

Comment:
I know this is test code, so arguably this doesn't matter, but shouldn't this be just `import java.util.Map`?